x/mint/keeper/emissions.go

```diff
@@ -0,0 +1,131 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/errors"
+	"cosmossdk.io/math"
+	"github.com/allora-network/allora-chain/x/mint/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// return the uncirculating supply, i.e. tokens on a vesting schedule
+// latest discussion on how these tokens should be handled lives in ORA-1111
+// probably thee tokens will be custodied off chain and this function will
+// just return the circulating supply based off of what the agreements off chain
+// were supposed to be at time of chain-genesis
+func GetLockedTokenSupply() math.Int {
+	return math.ZeroInt()
+}
+
+// helper function to get the number of staked tokens on the network
+// includes both tokens staked by cosmos validators (cosmos staking)
+// and tokens staked by reputers (allora staking)
+func GetNumStakedTokens(ctx context.Context, k Keeper) (math.Int, error) {
```

Comment:
are both of these tokens goveren'd by the same bonding time? Might be worth while noting the state of the token?

Reply:
I don't exactly know what you mean by bonding time in this context, but I do believe they have different withdrawal delay periods if that's what you're asking

I feel like making any decision about that is more of a paper question rather than code - i.e. go convince diederik